slo/slo.go

```diff
@@ -103,6 +104,14 @@ func (o Objective) AlertName() string {
 	return defaultAlertname
 }
 
+func (o Objective) AlertNameMetricAbsent() string {
+	if o.Alerting.Name != "" {
+		return o.Alerting.Name + "-" + defaultAlertnameAbsent
```

Comment:
Let's not concatenate the strings here. Let's keep the set value. It's either `SLOMetricAbsent` by default or whatever the user specifies directly.

Reply:
But this would be different from a ErrorBurnRate alert and should be an absent metric alert in this case. 🤔

Reply:
I think we want the contact for absent metric alert otherwise we can't distinguish between a customer named alert for error burn rate and one for absent metric.